src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,23 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        Book book = new Book();
+        book.setId(7L);
```

Comment:
Id must be generated by the database. Do not hardcode it in the code
don't hardcode ids, retrieve by getId()
```suggestion
```